composer.json

```
@@ -62,7 +62,7 @@
         "drupal/config_split": "2.0.1",
         "drupal/config_sync": "3.0.0-alpha3",
         "drupal/config_update": "2.0.0-alpha3",
-        "drupal/core-recommended": "10.2.7",
+        "drupal/core-recommended": "10.3.0-rc1",
```

10.3.0 is now available.
```suggestion
        "drupal/core-recommended": "10.3.0",
```